src/ui/base-page-storyEdit.js

```diff
@@ -62,6 +93,14 @@ https://raw.githubusercontent.com/fluid-project/sjrk-story-telling/master/LICENS
             setEditorDisplay: {
                 funcName: "sjrk.storyTelling.base.page.storyEdit.setEditorDisplay",
                 args: ["{that}.options.selectors.mainContainer", "{that}.options.selectors.pageContainer", "{that}.options.pageSetup.authoringEnabled", "{that}.options.pageSetup.hiddenEditorClass"]
+            },
+            showEditorHidePreviewer: {
```

Comment:
`showEditorHidePreviewer()` and `hideEditorShowPreviewer()` invokers can be combined into one by accepting a boolean argument.